pnpm-lock.yaml

```diff
@@ -16568,10 +16569,12 @@ packages:
       '@azure/msal-common': 14.12.0
       '@azure/msal-node-runtime': 0.13.6-alpha.0
       keytar: 7.9.0
+    dev: false
 
   /@azure/msal-node-runtime@0.13.6-alpha.0:
     resolution: {integrity: sha512-Tu5e4wBFiaiBLrOu7bsJF7FYknFH9XIOvUyCqCnmLJFMMOUnYULZ7fOFYuintFFcNPMOT2u8BVfSCPxETri5ng==}
     requiresBuild: true
+    dev: false
 
```

Comment:
Hopefully the myriad of `dev: false` additions aren't going to be removed by someone else's dep update.